src/pages/Home/index.js

```diff
@@ -1,23 +1,11 @@
-import { Redirect } from 'react-router-dom';
-
 import Word from 'components/Word';
-import useAuth from 'hooks/useAuth';
-import routesPaths from 'routes/routesPaths';
 
 import './styles.css';
 
-const Home = () => {
-  const { authenticated } = useAuth();
-
-  if (!authenticated) {
-    return <Redirect to={routesPaths.login} />;
-  }
```

Comment:
why did you delete this? we have all the authentication process, and if you are not logged in it should take you to the login page